modules/core/src/main/java/org/apache/ignite/internal/processors/cache/IgniteCacheOffheapManager.java

```diff
@@ -569,6 +569,23 @@ public void storeEntries(int partId, Iterator<GridCacheEntryInfo> infos,
      */
     public void dropRootPageForIndex(int cacheId, String idxName, int segment) throws IgniteCheckedException;
 
+    /**
+     * Renaming the root page of the index tree.
+     *
+     * @param cacheId Cache id.
+     * @param oldIdxName Old name of the index tree.
+     * @param newIdxName New name of the index tree.
+     * @param segment Segment number.
```

Comment:
Please clarify the comment. Segment index or something like that would be more clear.